### PR TITLE
ICMSLST-757 Add Textiles (Quota) response preparation view

### PR DESCRIPTION
**Pull request by mikeylockwood**
* Add view for response preparation for textiles
* Include link to edit goods using `GoodsTextilesLicenceForm`

### CODE REVIEW

File: web/templates/web/domains/case/import/manage/prepare-response.html [hunk 1]
```diff
@@ -3,6 +3,21 @@
...
+    <div class="three columns">
+      <label class="prompt west">{{ label }}</label>
+    </div>
+    <div class="six columns">
+      <span class="text-widget">{{ field or default }}</span>
+    </div>
+    <div class="three columns"></div>
+  </div>
+{%- endmacro %}
+{# END MACROS #}
```
Comment (kevinetienne) [inline web/templates/web/domains/case/import/manage/prepare-response.html:18]: Should we make the other [`application_field`](https://github.com/uktrade/icms/blob/bbb2ffb1cff786fbbc17fb200d65525dff670b5a/web/templates/web/domains/case/import/view_case.html#L17-L28) available more globally?

File: web/domains/case/_import/textiles/views.py [hunk 2]
```diff
@@ -249,3 +262,44 @@ def manage_checklist(request: AuthenticatedHttpRequest, *, application_pk: int)
             template_name="web/domains/case/import/management/checklist.html",
             context=context,
         )
+
+
+@login_required
+@permission_required("web.reference_data_access", raise_exception=True)
+def edit_goods_licence(request: AuthenticatedHttpRequest, application_pk: int) -> HttpResponse:
```
Comment (MattHolmes123) [inline web/domains/case/_import/textiles/views.py:269]: We have an unwritten rule about view arguments being keyword only (after the request) argument.

File: web/templates/web/domains/case/import/manage/prepare-response.html (hunk 1 above)
Comment (MattHolmes123) [reply web/templates/web/domains/case/import/manage/prepare-response.html:18]: We have form macros here:  
`icms/web/templates/forms/fields.html`  
What about a another file for displaying fields when viewing the application / screens like prepare-response? e.g.  
`icms/web/templates/display/fields.html`  
And then add application_section and application_field to there.

File: web/tests/domains/case/_import/textiles/test_admin_views.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import re
+
+import pytest
+from django.urls import reverse
+from pytest_django.asserts import assertRedirects
+
+from web.domains.case._import.textiles.models import TextilesApplication
+
```
Comment (MattHolmes123) [inline web/tests/domains/case/_import/textiles/test_admin_views.py:8]: Your tests are failing, I will review them when they are passing :+1:

**Review by MattHolmes123 — CHANGES_REQUESTED** [review]

File: web/templates/web/domains/case/import/manage/prepare-response.html (hunk 1 above)
Comment (mikeylockwood) [reply web/templates/web/domains/case/import/manage/prepare-response.html:18]: Yes that sounds sensible, will add

File: web/domains/case/_import/textiles/views.py (hunk 2 above)
Comment (mikeylockwood) [reply web/domains/case/_import/textiles/views.py:266]: I haven't seen this in any of the other import application views. Happy to change it, but we'd want all the views to be consistent

Comment (MattHolmes123) [reply web/domains/case/_import/textiles/views.py:266]: image

Comment (MattHolmes123) [reply web/domains/case/_import/textiles/views.py:266]: @mikeylockwood All of these are keyword only arguments after the request parameter.

**Review by oskusalerma — APPROVED** [review]

**Review by MattHolmes123 — APPROVED** [review]

**Pull request merged**